avocado/utils/pci.py

```diff
@@ -82,11 +82,11 @@ def get_num_interfaces_in_pci(dom_pci_address):
 
 def get_disks_in_pci_address(pci_address):
     """
-    Gets disks in a pci_address.
+    Gets disks in a PCI address.
```

Comment:
The structured correlation to the `pci_address` variable should be done with the proper Sphinx statements, such as `:param pci_address:`.  For humans, I think we should use proper English.